packages/conf-numa/conf-numa.0.1.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Steve Bleazard <stevebleazard@googlemail.com>"
+authors: "Steve Bleazard <stevebleazard@googlemail.com>"
+homepage: "https://www.github.com/stevebleazard/ocaml-conf-numa"
+bug-reports: "https://www.github.com/stevebleazard/ocaml-conf-numa/issues"
+license: "MIT"
+dev-repo: "https://www.github.com/stevebleazard/ocaml-conf-numa.git"
+
+build: [
+  ["sh" "-exc" "cc -c $CFLAGS -I/usr/local/include files/test.c"]
```

Comment:
I don't see `files/test.c` in this pull request, did you forget to `git add` it?